cmd/commands/cli/command.go

```diff
@@ -332,7 +332,7 @@ func (c *cliApp) help() {
 
 // quit stops cli and client commands and exits application
 func (c *cliApp) quit() {
-	stop := utils.SoftKiller(c.Kill)
+	stop := utils.HardKiller(c.Kill)
```

Comment:
`HardKiller` doing `os.Exit()` without proper shutting down other dependencies. 
I think there should be a better approach for this.

Reply:
CLI is starting [here](https://github.com/mysteriumnetwork/node/blob/master/cmd/commands/cli/command.go#L57), but if we are doing `os.Exit()` in the CLI we are missing the [following](https://github.com/mysteriumnetwork/node/blob/master/cmd/commands/cli/command.go#L63-L65) block. 
It's not only about CLI, but it's also about upper components.